sky/engine/core/dart/painting.dart

```diff
@@ -304,6 +310,7 @@ class Canvas extends NativeFieldWrapperClass2 {
   }
 }
 
+/// An opaque object representing a sequence of recorded drawing commands.
```

Comment:
It might be confusing whether this means opaque as in alpha=0xFF or opaque as in "cannot be inspected"